### ISSUE TITLE
Standardize function and variable naming

### ISSUE THREAD

**Issue by globusharris** [IQSS/PSI-Library#48]
Variable and function names are currently a combination of camelCase and delimiter separated with period (e.g. function.name). We should pick one and go with it before making any extensions at least, can change old code as we go through the code review (how easy this is to do should also be a good indicator of whether or not the code is adequately modular). I personally vote camelCase since the period makes it seem like object attributes for those less familiar with R, but wanted to check before making changes in case R has some standard naming conventions I am unaware of. 

(Also could use underscore delimiters or something, but since half the variables are already camelCase this seems easiest?)

**tldr: if you have strong opinions about what our variable naming convention should be, speak now or forever hold your peace.**
